docs/tutorials/data-loading/remote-files.md

```diff
@@ -15,82 +13,190 @@ You need the following software installed:
-service-defined HTTP endpoints.
+The S3 File Connector example is a connector built to display how to be able to download and upload files
+to and from an S3 bucket in AWS.
+The connector is built in Node.js and uses the gRPC protocol to stream data to and from the Qlik Associative Engine.
 
 ## Setup
 
 To run the example code, clone the
-[qlik-oss/core-file-connectivity-service](https://github.com/qlik-oss/core-file-connectivity-service) Git repository.
+[qlik-oss/core-grpc-s3-file-connector](https://github.com/qlik-oss/core-grpc-s3-file-connector) Git repository.
 Before you continue, look at the documentation to get familiar with the content and structure of
 the repository.
 
 !!!Note
     In your shell, make sure current directory is at the repository root.
 
-First, install the dependencies:
+The connector example repo provides a `docker-compose.yml` file
+that starts the example connector together with a Qlik Associative Engine.
+
+```yml
+version: "3.3"
+
+services:
+  qix-engine:
+    image: qlikcore/engine:12.350.0
+    ports:
+      - 9076:9076
+    # EnableGrpcFileStreamConnector parameter enables the feature.
+    # Default is disabled (0)
+    # GrpcConnectorPlugins parameter configures the connector in engine.
+    # Format <connector name>,<host>:<port>.
+    # The "fs" at the end of the string marks that the connector
+    # supports streaming of files
+    command: -S EnableGrpcFileStreamConnector=1 -S GrpcConnectorPlugins="s3-grpc-file-connector,s3-grpc-file-connector:50051,fs" -S AcceptEULA=${ACCEPT_EULA}
+  s3-grpc-file-connector:
+    build: ../ # Build the connector from the Dockerfile
+    ports:
+      - 50051:50051 # Expose the connector on port 50051
+    environment:
+      # The following variables are passed as environment variables to the connector upon startup
+      CORE_S3_FILE_CONNECTOR_BUCKET_NAME: ${CORE_S3_FILE_CONNECTOR_BUCKET_NAME} # Name of the S3 bucket that the connector should target
+      CORE_S3_FILE_CONNECTOR_BUCKET_ACCESS_KEY_ID: ${CORE_S3_FILE_CONNECTOR_BUCKET_ACCESS_KEY_ID} # Access key id for the S3 bucket
+      CORE_S3_FILE_CONNECTOR_BUCKET_SECRET_ACCESS_KEY: ${CORE_S3_FILE_CONNECTOR_BUCKET_SECRET_ACCESS_KEY} # Access key token for the S3 bucket
+```
 
-```sh
-npm install
+In this example we are going to use a data set from [`airports.csv`](https://github.com/qlik-oss/core-grpc-s3-file-connector/blob/master/examples/airports.csv).
+This data set is present in an S3 Bucket in AWS.
+The name of the S3 bucket is configured in the example connector by passing it as an environment variable; `CORE_S3_FILE_CONNECTOR_BUCKET_NAME`.
+To be able to access the S3 bucket the connector will also need to provide the credentials,
+which in this example is the access key id `CORE_S3_FILE_CONNECTOR_BUCKET_ACCESS_KEY_ID`
+and a secret `CORE_S3_FILE_CONNECTOR_BUCKET_SECRET_ACCESS_KEY`.
+
+To run this example you will need to set up your own S3 bucket and configure the name and credentials as described above.
```

Comment:
It might be a bit unclear that above refers to the instructions inside the linked yaml file.